src/ttl_table.h

```diff
@@ -31,4 +35,12 @@ bool TimeToLiveTable_HasExpiration(TimeToLiveTable *table, t_docId docId);
 bool TimeToLiveTable_HasDocExpired(TimeToLiveTable *table, t_docId docId, const struct timespec* expirationPoint);
 bool TimeToLiveTable_VerifyDocAndFields(TimeToLiveTable *table, t_docId docId, const t_fieldIndex* sortedFieldIndices, size_t fieldCount, enum FieldExpirationPredicate predicate, const struct timespec* expirationPoint);
 
+bool TimeToLiveTable_VerifyDocAndField(TimeToLiveTable *table, t_docId docId, t_fieldIndex field, enum FieldExpirationPredicate predicate, const struct timespec* expirationPoint);
```

Comment:
what does it differ from the function just above?

Reply:
The above function will eventually be removed. It gets an array of `t_fieldIndex` while this one gets a single index